src/serialization/sb2.js

```diff
@@ -1061,7 +1061,8 @@ const parseBlock = function (sb2block, addBroadcastMsg, getVariableId, extension
             children: [],
             mutation: {
                 tagName: 'mutation',
-                proccode: procData[0], // e.g., "abc %n %b %s"
+                proccode: procData[0].replace(/\\%/g, '％').replace(/\\(.)/g, '$1'),
```

Comment:
Note that this is a *fullwidth* percent sign-- "replace this character with another very similar looking character" strikes me as somewhat janky